auth.py

```
@@ -8,7 +8,7 @@
 from urllib.parse import urlparse  # werkzeug.urls.url_parseから標準ライブラリに変更
 from models import db, User
 from forms import LoginForm, RegistrationForm
-from services import UserService
+from service_layer import UserService
```

Ruff (0.12.2)

`service_layer.UserService` imported but unused

Remove unused import: `service_layer.UserService`

(F401)